src/redux/rockets/rockets.js

```diff
@@ -1,6 +1,8 @@
 const initialState = [];
 
 const FETCH_ROCKET_DATA = 'space-travelers-hub/rockets/FETCH_ROCKET_DATA';
+const RESERVE_ROCKET = 'space-travelers-hub/rockets/RESERVE_ROCKET';
+const CANCEL_RESERVATION = 'space-travelers-hub/rockets/CANCEL_RESERVATION';
 
 export const getRocketsData = () => async (dispatch) => {
   const response = await fetch('https://api.spacexdata.com/v3/rockets', {
```

Comment:
[optional] A good practice would be adding error handling in your fetch API function, for example, implementing a `try() catch()`, so if there is a network failure or a server down, your app can handle those errors and display them properly on the page, because we are using external data and we can't foresee any future issues on those services.